### PR TITLE
Add updated benefits guidelines

### PR DESCRIPTION
**Pull request by damianavila**
OK, this is the updated benefits section after all the discussion happening at https://github.com/2i2c-org/meta/issues/444.

For details about the reasons behind this proposal, please check this message: https://github.com/2i2c-org/meta/issues/444#issuecomment-1463965722.

### CODE REVIEW

File: people/compensation.md [hunk 1]
```diff
@@ -64,5 +63,58 @@ We have [internal salary documents for the specific positions and levels](https:
...
+
+### Time off
+
+Our time off policy is documented in our [Taking time off](./time-off.md) section.
+
+### Mechanism for payment
+
+For anything that involves purchasing equipment, services, tickets, etc, we have these options:
+
+* Employee team members can pay via their own methods, and request a reimbursement via Ramp.
```
Comment (pnasrat) [inline people/compensation.md:80]: Link to https://compass.2i2c.org/en/latest/administration/reimburse.html as source of truth as for reimbursements for employees it's via invoice not ramp I believe.

**Review by pnasrat — CHANGES_REQUESTED** [review]
@choldgraf recently added clarification about expense reimbursement - please link to that as the source of truth in that section

**Comment by jmunroe** [comment]
I think these are great set of policies and clearly written.

Do we need to state an 'effective date' for this revision of the policy?  I think we said wanted to make this effective as January 1, 2023.

Since the policy mentions 'yearly' amounts, should we clarify that these benefits do not roll over between calendar years.  While I would encourage 2i2c members to utilize these benefits to help them with both their work and wellness, I don't we want us to be accumulating any liability for unpaid benefits year over year.  Do we need to put in a deadline for benefits claims for the previous calendar year (somthing like March 31)?  

How should we handle benefits for team members starting mid way through the calendar year? Are we pro-rating the benefits?  

(None of these are blocking issues for accepting the PR but may need to be addressed in a future version.)

File: people/compensation.md [hunk 2]
```diff
@@ -2,16 +2,15 @@
...
-See that document for ideas that are more specific / structured than this page.
-:::
-
 :::{admonition} Aspirational
 We are a young organization, and this page describes the salary policy that we aim to enact in the coming years.
 It may not accurately reflect our current practices, but we share it for transparency and to provide a goal for our team to shoot for.
 :::
 
+:::{admonition}
+We are working on an update of our existing compensation policy. More details about the upcoming changes will be updated here as soon we we reached consensus and approval for the new policy.
```
Comment (sgibson91) [inline people/compensation.md:11]:
```suggestion
We are working on an update of our existing compensation policy. More details about the upcoming changes will be updated here as soon we reach consensus and approval for the new policy.
```

File: people/compensation.md [hunk 3]
```diff
@@ -64,5 +63,58 @@ We have [internal salary documents for the specific positions and levels](https:
 
 ## Benefits
 
-Benefits are offered by {term}`Code for Science and Society`.
-They are offered via [TriNet](https://www.trinet.com/) as described in [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).
+This section describes our benefits policy across all team members at 2i2c. 2i2c team members include people who are designated as employees and as contractors, depending on their role at 2i2c.
+
+Benefits are available to employees only and are managed through {term}`Code for Science and Society`’s relationship with a service partner or PEO. For example, employees who are US based and employed through {term}`Code for Science and Society`’s relationship with [Trinet](https://www.trinet.com/), will have access to health coverage and retirement benefits through Trinet. Employees based outside the US and employed through a relationship with a PEO like Remote will have access to benefits available in their region through that PEO. More details can be found at the [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).
```
Comment (sgibson91) [inline people/compensation.md:68]: What about contractors who have no relationship with a PEO?

File: people/compensation.md [hunk 4]
```diff
@@ -64,5 +63,58 @@ We have [internal salary documents for the specific positions and levels](https:
...
+
+Contractors are independent, not employed by CS&S, and are brought on to support 2i2c with their specific expertise. To support these contract relationships, 2i2c will cover some types of expenses related to the operation of independent contract business, such as would typically be changed by overhead at a larger organization. These expenses include healthcare, accounting services, and reimbursable equipment expenses. We estimate a 20% on top of the contractor monthly payment to cover for these sort of expenses.
```
Comment (sgibson91) [inline people/compensation.md:70]:
```suggestion
Contractors are independent, not employed by CS&S, and are brought on to support 2i2c with their specific expertise. To support these contract relationships, 2i2c will cover some types of expenses related to the operation of independent contract business, such as would typically be charged as overhead at a larger organization. These expenses include healthcare, accounting services, and reimbursable equipment expenses. We estimate a 20% on top of the contractor monthly payment to cover for these sort of expenses.
```

I think? Either way, this last phrase didn't parse correctly to me

File: people/compensation.md [hunk 5]
```diff
@@ -64,5 +63,58 @@ We have [internal salary documents for the specific positions and levels](https:
...
+### Travel
+
+2i2c will reimburse any work-related travel expenses for its team members.
+Team members should get approval for travel (and rough budget) before any booking.
+
+### Personal development
+
+2i2c team members should identify opportunities to learn and improve the skills that are related to their role and their learning goals.
+2i2c will cover the cost of these opportunities.
+We particularly encourage area leads and managers to identify these opportunities for others in their area (and for themselves)
```
Comment (sgibson91) [inline people/compensation.md:105]:
```suggestion
We particularly encourage area leads and managers to identify these opportunities for others in their area (and for themselves).
```

Missing punctuation

**Review by sgibson91 — COMMENTED** [review]
Thanks for writing this up @damianavila! I have left a few suggestions that are basically typos. I won't explicitly approve yet as I believe Pris and James have raised good points.

File: people/compensation.md (hunk 1 above)
Comment (damianavila) [reply people/compensation.md]: Updated! Thanks for the reminder!

**Comment by damianavila** [comment]
Replying to @jmunroe's comments:

* Effective day: **January 1st, 2023**
* Should we clarify that these benefits do not roll over between calendar years? YES
* Do we need to put in a deadline for benefits claims for the previous calendar year (something like March 31)? March 31st is a sensible choice, IMHO.
* Are we pro-rating the benefits? Mmm, good one... pro-rating seems a good enough solution, I would say.

I added notes about all of these in the [parent issue]() and this PR as well (as clarification in an admonition)!

File: people/compensation.md (hunk 3 above)
Comment (damianavila) [reply people/compensation.md:68]: Replied here: https://github.com/2i2c-org/meta/issues/444#issuecomment-1466964255

**Comment by damianavila** [comment]
@pnasrat @jmunroe and @sgibson91, also cc the rest of @2i2c-org/engineering for another look after incorporating your feedback.

File: people/compensation.md [hunk 6]
```diff
@@ -64,5 +63,59 @@ We have [internal salary documents for the specific positions and levels](https:
...
+
+Contractors are independent, not employed by CS&S, and are brought on to support 2i2c with their specific expertise. To support these contract relationships, 2i2c will cover some types of expenses related to the operation of independent contract business, such as would typically be charged as overhead at a larger organization. These expenses include healthcare, accounting services, and reimbursable equipment expenses. We estimate a 20% on top of the contractor monthly payment to cover for these sort of expenses.
```
Comment (GeorgianaElena) [inline people/compensation.md:70]: Hmm, I think this this should be a bit rephrased to match the info in https://github.com/2i2c-org/meta/issues/444#issuecomment-1466964255. After reading it, it seemed like _"expenses include healthcare, accounting services, and reimbursable equipment expenses"_ are reimbursed up to 20%.

Maybe something like the following:
```suggestion
Contractors are independent and not employed by CS&S or a PEO. They are brought on to support 2i2c with their specific expertise. They are not employees, so employees benefits are not available in this situation. 2i2c estimates that the employees specific benefits account for 20% of a monthly payment. In order to support role equity, this amount will be treated as overhead and will be added on top of the contractor monthly payment.
```

File: people/compensation.md [hunk 7]
```diff
@@ -64,5 +63,59 @@ We have [internal salary documents for the specific positions and levels](https:
 
 ## Benefits
 
-Benefits are offered by {term}`Code for Science and Society`.
-They are offered via [TriNet](https://www.trinet.com/) as described in [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).
+This section describes our benefits policy across all team members at 2i2c. 2i2c team members include people who are designated as employees and as contractors, depending on their role at 2i2c.
+
+Benefits are available to employees only and are managed through {term}`Code for Science and Society`’s relationship with a service partner or PEO. For example, employees who are US based and employed through {term}`Code for Science and Society`’s relationship with [Trinet](https://www.trinet.com/), will have access to health coverage and retirement benefits through Trinet. Employees based outside the US and employed through a relationship with a PEO like Remote will have access to benefits available in their region through that PEO. More details can be found at the [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).
```
Comment (GeorgianaElena) [inline people/compensation.md:68]: Can we group these benefits into a category of some sort? maybe payroll? (no idea if this is correct)

The reasoning is that we have this big section called _**"Benefits"**_ that describes the overall policy for employees & contractors. 

But then, there's the following sentence _"Benefits are available to employees only"_ which makes it a bit unclear that the extras defined in their own section below are available to everyone and not to employees only.

If we'd find a category for this, let's call it _payroll_ (again, I'm not sure this is correct), something like _"Payroll benefits are available to employees only"_ would be more clear that it refers to its own category of benefits.

What do you think?

**Comment by jmunroe** [comment]
Thanks! I'm fine with merging these changes.

**Comment by pnasrat** [comment]
LGTM

File: people/compensation.md (hunk 7 above)
Comment (damianavila) [reply people/compensation.md:68]: I restructure things a little bit more in my [latest commit](https://github.com/2i2c-org/team-compass/pull/682/commits/1cf40ee0d209e80db3e7c416e93a21b9da85ead9), let me know if that makes more sense to you!

File: people/compensation.md (hunk 6 above)
Comment (damianavila) [reply people/compensation.md]: I am not accepting the proposed changes, please check my comment on the parent issue for more details: https://github.com/2i2c-org/meta/issues/444#issuecomment-1469031926.

File: people/compensation.md (hunk 7 above)
Comment (damianavila) [reply people/compensation.md:68]: Resolving for now, but feel free to unresolve if you disagree.

**Comment by damianavila** [comment]
@2i2c-org/engineering, I changed things a little bit after @GeorgianaElena's feedback, another look is welcome 😉!

**Review by GeorgianaElena — APPROVED** [review]
Thanks @damianavila!

**Review by colliand — APPROVED** [review]
These changes look good to me. Thanks @damianavila for this contribution and thanks to everyone who provided feedback.

**Comment by damianavila** [comment]
Finally merged, thanks to all who contributed with their thoughts and feedback!!
Please check the parent issue for the next steps to operationalize the decisions we made.

**Pull request merged**